docs/acp/user/developing-an-app/developing-an-app.md

```diff
@@ -28,130 +38,201 @@ You need to have a clear understanding of what your app API will provide to othe
 
-You can skip through the prompts by hitting **Enter**.
+## Adding App manifest files
 
-{% endinfo_block %}
+Before your App can be listed in the App Store Catalog you need to add the following files.
 
-Starting from [Spryker SDK](/docs/sdk/dev/spryker-sdk.html) version 0.3.0, you can use the following set of commands:
+### Manifest
 
-```bash
-cd /www/my-app # The local project directory you created before
-spryker-sdk sdk:workflow:run # and select the app workflow
-```
-The workflow guides you as much as possible through the process of building an app. Whenever something needs manual interaction, the workflow stops with a message on what you need to do. After you completed the manual step, re-run the workflow with the `spryker-sdk sdk:workflow:run` command. This continues the previously paused workflow.
+The manifest file is the most important one for the App and contains data that will be displayed in the App Store Catalog. You can use the [manifest code snippet](/docs/acp/user/developing-an-app/code-snippets/manifest.html) and update it to your needs. Add this file to `config/app/manifest/en_US.json` of your App.
 
-### Workflow details
+Manifest files have to have the local name as file name e.g. `en_US.json` and the file has to be placed inside the `config/app/manifest` directory.
 
-The `spryker-sdk sdk:workflow:run` command guides you through the whole process of building an app. This command does the following.
+### Configuration
 
-#### 1. Downloads the boilerplate code and adds app definition files
+This file contains all necessary form fields for inputs required by the user of your App to be displayed in the App Store Catalog. You can use the [configuration code snippet](/docs/acp/user/developing-an-app/code-snippets/configuration.html) and update it to your needs. Add this file to `config/app/configuration.json` of your App.
 
-One of the first steps of the workflow is downloading a reduced Spryker project into the working directory. It already contains a lot of code that lets you start implementing your business logic.
-This step can take a few minutes to complete.
+### Translation
 
-{% info_block infoBox "Info" %}
+This file contains all translations for the form fields you’ve previously defined. You can use the Hello World example translation file and update it to your needs. Add this file to `config/app/translation.json` of your App.
 
-At this point, no dependencies are installed. 
+## Adding the Registry code
 
-{% endinfo_block %}
+Every App needs some default endpoints for the App Registry Service. This service is in between of the App Store Catalog and all Apps. Each App will be registered in the App Registry Service and to be able to configure your App needs the following endpoints for communication:
+ - Configure
+ - Disconnect
 
-There are a couple of files that an app must have. The workflow guides you through creating them.
+To be able to use your Mini Framework as an App you need to add a new Spryker module:
```

Comment:
Can they be pre-installed?

Reply:
It is preinstalled in the final example 
See `Download the completed example from the Mini-Framework`